spec/requests/users_spec.rb

```diff
@@ -1,7 +1,23 @@
 require 'rails_helper'
 
-RSpec.describe 'Users', type: :request do
-  describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+RSpec.describe UsersController, type: :request do
+  before(:all) do
+    @first_user = User.create(name: 'happen', photo: 'photo', bio: 'test')
+  end
+
+  describe 'GET index' do
+    it 'renders index' do
+      get :index
+      expect(response.status).to eq(200)
+      expect(response).to render_template('index')
```

Comment:
- [ ] Great job so far.  please kindly include tests that ensures the response body includes the correct placeholder text.
Example : `expect(response.body).to include("Welcome to user index page")`

Reply:
Hi @tufoinnkuo10 

Thank you for your review, This has been resolved